examples/meta/src/converter/converter_kernel_locally_linear_embedding.sg.in

```diff
@@ -0,0 +1,18 @@
+File f_feats_train = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
+
+#![create_features]
+Features feats = create_features(f_feats_train)
+#![create_features]
+
+#![set_parameters]
+Kernel k = create_kernel("LinearKernel", width=1.0)
```

Comment:
https://dev.azure.com/shogunml/shogun/_build/results?buildId=3926&view=logs&j=be3b03d7-4089-5ab3-9462-9af4d8b1dcdf&t=92f9e07a-f827-5326-7f00-9de226b0894b&l=336

Same thing, please run stuff locally before you send the PR.